config/application.yml.default

```diff
@@ -586,7 +584,7 @@ test:
   scrypt_cost: 800$8$1$
   secret_key_base: test_secret_key_base
   session_encryption_key: 27bad3c25711099429c1afdfd1890910f3b59f5a4faec1c85e945cb8b02b02f261ba501d99cfbb4fab394e0102de6fecf8ffe260f322f610db3e96b2a775c120
-  short_term_phone_otp_rate_limiter_enabled: false
+  short_term_phone_otp_max_attempts: 100
```

Comment:
Maybe we should have something which resets all rate limits between tests? Unclear if that would actually help for the reason this was added.

That, and/or use a [fake phone generator](https://github.com/faker-ruby/faker/blob/main/doc/default/phone_number.md) in [the phone factory setup](https://github.com/18F/identity-idp/blob/8074c008ff15458aa4317cc77ad6b6d95948b60f/spec/factories/phone_configurations.rb#L6).

Reply:
I think some of them were sending too many SMS within the same test. ex: https://github.com/18F/identity-idp/blob/333c22c23ab5eb1d8d305d085e3f7db796137d8f/spec/features/openid_connect/openid_connect_spec.rb#L33-L38

Reply:
Hm, fair enough. Seems like we may want to revisit those at some point to make them more realistic to what we'd expect in the real-world.

I started poking at my previous idea of resetting the rate limiter, though I don't love the idea here of multiple calls to Redis between each test:

```rb
# spec/support/rate_limiter.rb

RSpec.configure do |config|
  config.before(:each) do
    REDIS_THROTTLE_POOL.with do |client|
      keys = client.call [:keys, 'throttle:*']
      client.call [:del, *keys]
    end
  end
end
```

Reply:
do we need the `.call`? I think we can:

```ruby
RSpec.configure do |config|
  config.before(:each) do
    REDIS_THROTTLE_POOL.with do |client|
      keys = client.keys('throttle:*')
      client.del(*keys)
    end
  end
end
```

Reply:
Stumbling my way through the solution led me to this issue, which seemed to imply that `call` is a bit safer when deleting many keys. Though I doubt we have so many matches that it'd become an issue.

https://github.com/redis/redis-rb/issues/122

Reply:
I think we intend to reset the rate limit pool between tests: https://github.com/18F/identity-idp/blob/333c22c23ab5eb1d8d305d085e3f7db796137d8f/spec/rails_helper.rb#L112

Reply:
Ah, that would (likely) work too! As you mentioned, it's probably more to do with limits within a single test being exceeded.